python/tflite_micro/BUILD

```diff
@@ -1,3 +1,5 @@
+load("@rules_python//python:defs.bzl", "py_library")
```

Comment:
The two loads can be in a single line. Here and in the rest of the PR.

Reply:
Done !